intelmq/bots/BOTS

```diff
@@ -826,6 +826,18 @@
                 "port": "<port>"
             }
         },
+        "UDP": {
+            "description": "UDP is a simple UDP bot responsible to send events to a udp port (e.g.: syslog daemon).",
+            "module": "intelmq.bots.outputs.udp.output",
+            "parameters": {
+                "field_delimiter": "|",
+                "format": "<json/delimited> read README.md",
```

Comment:
s/read README.md/for more explanations about this field, checkout out the README.md"